### PR TITLE
Adds Feed Page and Button Prototype

### PR DESCRIPTION
**Pull request by ubahl**
Small PR: adds an HTML page for the feed of posts and a button prototype at the top.

Screenshot 2020-07-24 at 11 58 31 AM

### CODE REVIEW

File: src/main/webapp/feed.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="feedStyle.css">
+    <link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
+    <script src="script.js"></script>
+    
+  </head>
+  <body>
+      <button id="create-post-button" type="button">create a post</button>
+  </body>
+</html>
```
Comment (mirrorkeydev) [inline src/main/webapp/feed.html:15]: newline :)

File: src/main/webapp/feedStyle.css [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  font-size: 18px;
+  font-weight: 400;
+  height: 50px;
+  margin-left: 50%;
+  margin-right: auto;
+  min-width: 100px;
+  outline: none;
+  text-align: center;
+  transform: translate(-50%);
+  width: 50%;
```
Comment (mirrorkeydev) [inline src/main/webapp/feedStyle.css:15]: looks great! alphabetizing looks so nice when there's lots of attributes haha

File: src/main/webapp/feed.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8"/>
+    <title>Feed</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="feedStyle.css">
+    <link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
+    <script src="script.js"></script>
+
```
Comment (areetaw) [inline src/main/webapp/feed.html:10]: Extra line not needed!

**Comment by areetaw** [comment]
In the master branch, I included a find-events.html for you as well as included all of the styling needed so make sure to pull, otherwise everything LGTM!

File: src/main/webapp/feedStyle.css [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+}
```
Comment (areetaw) [inline src/main/webapp/feedStyle.css:16]: newline for files!

Comment (ubahl) [reply src/main/webapp/feedStyle.css:16]: There is a newline there, it'll show the red icon when there isn't

File: src/main/webapp/feed.html (hunk 3 above)
Comment (ubahl) [reply src/main/webapp/feed.html]: Thanks!

File: src/main/webapp/feedStyle.css (hunk 2 above)
Comment (ubahl) [reply src/main/webapp/feedStyle.css:15]: Haha thanks!

File: src/main/webapp/feed.html (hunk 1 above)
Comment (ubahl) [reply src/main/webapp/feed.html]: Ooh thanks!

**Pull request merged**